transaction-server/database/user_database.go

```diff
@@ -19,22 +18,27 @@ type UserDatabase interface {
 	GetFunds(string) (decimal.Decimal, error)
 	RemoveFunds(string, decimal.Decimal) error
 
-	AddStock(user string, stock string, shares int) error
-	GetStock(user string, stock string) (int, error)
-	RemoveStock(user string, stock string, amount int) error
+	AddStock(user string, stock string, shares decimal.Decimal) error
```

Comment:
Just curious why did these need to change?

Reply:
In the log page, the amount examples were listed as decimals: http://www.ece.uvic.ca/~seng462/ProjectWebSite/Commands.html

I know for the purpose of this he mentioned we don't have to worry about partial stocks, but I figured might as well make them all decimals. That way we also don't have to worry about int vs decimal types too.